var/spack/repos/builtin/packages/hoomd-blue/package.py

```diff
@@ -51,6 +51,7 @@ class HoomdBlue(CMakePackage):
     # clone a specific tag from Bitbucket instead of using the tarballs.
     # https://bitbucket.org/glotzer/hoomd-blue/issues/238
     version('2.1.6', git=git, tag='v2.1.6', submodules=True)
+    version('2.2.2', git=git, tag='v2.2.2', submodules=True)
```

Comment:
Can you keep versions sorted from newer to older?

Reply:
@brockpalen Can you please do the change above and then ping me?